docs/internal/dev/versioning.md

```
@@ -89,21 +89,20 @@ npx changeset add --empty  # Creates an empty changeset (for chore/docs changes)
 
 ### Automated Pipeline (CI)
 
-The full release flow is automated via `auto-release.yml`, triggered automatically when a `staging→main` PR merges:
+The full release flow is automated via `auto-release.yml`, triggered automatically on every push to `main`:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)